docs/source/gen/flytectl_create_project.rst

```diff
@@ -0,0 +1,98 @@
+.. _flytectl_create_project:
+
+flytectl create project
+-----------------------
+
+Create project resources
+
+Synopsis
+~~~~~~~~
+
+
+
+Create the projects.(project,projects can be used interchangeably in these commands)
+
+::
+
+ bin/flytectl create project --name flytesnacks --id flytesnacks --description "flytesnacks description"  --labels app=flyte
+
+Create Project by defination file
+::
+
+ bin/flytectl create project --file project.yaml 
+
+.. code-block:: yaml
+
+   id: "project-unique-id"
+   name: "Friendly name"
+   labels:
+	  app: flyte
```

Comment:
Can we check if there are some whitespaces/tabs causing this

Reply:
no there is no whitespace